src/components/Display.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Button extends React.Component {
+  render() {
+    const { name } = this.props;
+    return <button type="submit">{name}</button>;
+  }
+}
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+};
```

Comment:
- maybe you want to call this prop `result` as specified by [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png) 

- default value of this prop should be `'0'`, enforce this with `defaultProps` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)